ArcGISRuntimeSDKQt_CppSamples/Layers/ExportTiles/ExportTiles.cpp

```diff
@@ -56,22 +56,33 @@ void ExportTiles::componentComplete()
   m_mapView = findChild<MapQuickView*>("mapView");
 
   // create a tiled basemap
-  ArcGISTiledLayer* tiledLayer = new ArcGISTiledLayer(m_serviceUrl, this);
-  Basemap* basemap = new Basemap(tiledLayer, this);
+  Basemap* basemap = new Basemap(BasemapStyle::ArcGISImagery, this);
+
+  // create an export tile cache task when basemap has finished loading
+  connect(basemap, &Basemap::doneLoading, this, [this]()
+  {
+    if (m_map->basemap()->baseLayers()->size() > 0)
```

Comment:
I suggest using isEmpty instead - one less comparison to make - https://developers.arcgis.com/qt/cpp/api-reference/esri-arcgisruntime-layerlistmodel.html#isEmpty